File: Zadanie 5/MVVM/MVVM/Model/Author.cs
namespace MVVM.Model
{
    using System.Xml.Serialization;

    public class Author
    {
        [XmlAttribute("authorID")]
        public string AuthorID { get; set; }

        [XmlElement("lastname")]
        public string LastName { get; set; }

        [XmlElement("firstname")]
        public string FirstName { get; set; }
    }
}

File: MVC/src/main/java/Model/Author.java
package Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor
public class Author {

    private String AuthorID;

    private String LastName;

    private String FirstName;
}
